extensions/git/src/git.ts

```diff
@@ -1536,6 +1536,25 @@ export class Repository {
 		await this.exec(args);
 	}
 
+	async fastForwardBranch(options: { remote: string; ref: string; readonly cancellationToken?: CancellationToken }): Promise<void> {
```

Comment:
Since this is basically `fetch`, shouldn't it be just additional options on the existing `fetch` command?

Reply:
Switched to using the `fetch()` method.

Reply:
I meant something else. Instead of creating a `fastForwardBranch()` method, should this behavior just be added to the existing `fetch()` method, behind additional options?

Reply:
`fastForwardBranch()` has been removed.